base_phone/README.rst

```diff
@@ -0,0 +1,99 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==========
+Base Phone
+==========
+
+This module validate phone numbers using the *phonenumbers* Python library,
+which is a port of the library used in Android smartphones. For example, if
+your user is linked to a French company and you update the form view of a
+partner with a badly written French phone number such as '01-55-42-12-42',
+Odoo will automatically update the phone number to E.164 format '+33155421242'
+and display in the form and tree view of the partner the readable equivalent
+'+33 1 55 42 12 42'.
+
+This module also adds *tel:* links on phone numbers and *fax:* links on fax
```

Comment:
this text seems obsolete. tel link seems native in recent odoo version. Don't you think ?